doc/runtimes/hugo.md

```diff
@@ -12,7 +12,7 @@ details on components and configuration of build images.
 
 The Hugo toolset is run when one of following conditions met:
 
-1. `config.yaml`, `config.toml`, `config.yml` or `config.json` configuration files in root of repo.
+1. `config/hugo.yaml`, `config/hugo.toml`, `config/hugo.yml` or `config/hugo.json` configuration files in root of repo.
```

Comment:
I actually think we should word this line differently as users may interpret the `config` part of each file type as the directory name, especially because our second bullet below mentions we search the `config` directory recursively.

I'd be in favor of writing this as something like the following:

_1. Any supported `config.*` files at the root of the repository: `config.toml`, `config.yaml`, `config.yml`, `config.json`_
_2. Any supported `hugo.*` files at the root of the repository: `hugo.toml`, `hugo.yaml`, `hugo.yml`, `hugo.json`_
_3. Recursively look up supported files in the `config` directory: `config/**/*.toml`, `config/**/*.toml`
  `config/**/*.yml`, `config/**/*.json`_